Script/test-SPM.sh

```diff
@@ -30,7 +30,7 @@ let package = Package(
     name: \"TestProject\",
     defaultLocalization: \"en-US\",
     platforms: [
-        .iOS(.v10)
+        .iOS(.v11)
```

Comment:
I suppose you missed updating test-pospec script 
https://github.com/adobe/aepsdk-campaignclassic-ios/blob/dev/Script/test-podspec.sh#L28

Reply:
This is not yet resolved ⏫